src/point.js

```diff
@@ -1,13 +1,15 @@
 export default function(node, event) {
-  var svg = node.ownerSVGElement || node;
+    var svg = node.ownerSVGElement || node;
 
-  if (svg.createSVGPoint) {
-    var point = svg.createSVGPoint();
-    point.x = event.clientX, point.y = event.clientY;
-    point = point.matrixTransform(node.getScreenCTM().inverse());
-    return [point.x, point.y];
-  }
+    if (svg.createSVGPoint) {
+        var point = svg.createSVGPoint();
+        if (event) {
```

Comment:
It's not clear how this `event` would end up as undefined here. Is there an example you could share that reproduces the error you see?